package.json

```diff
@@ -149,17 +151,18 @@
     "react-sortable-hoc": "^2.0.0",
     "rimraf": "^3.0.2",
     "semver": "^7.3.5",
+    "size-limit": "^8.0.1",
     "style-loader": "^2.0.0",
     "turbo": "^1.0.6",
     "typescript": "4.7.3"
   },
   "engines": {
     "node": ">=14.17.0",
-    "yarn": ">=1.19.0"
+    "yarn": ">=1.18.0"
```

Comment:
downgrade, more compatibility 👍🏼

Reply:
I'm not sure why we required 1.19 in the first place, any idea?